packages/ui-inputs/src/button.tsx

```diff
@@ -1,15 +1,218 @@
-import * as React from 'react'
-import { Button as AntButton } from 'antd';
-
+import * as React from 'react';
+import {useTheme, createUseStyles, theming} from './theme';
+import {Button as AntButton} from 'antd';
+import { ReactNode } from 'react';
+import { SpinnerBright24Icon, SpinnerDark24Icon } from '@kaltura-path/ui-icons';
+const classNames = require('classnames');
 
 export interface ButtonProps {
-  label: string;
-  onClick: () => void
+    label?: string;
+    disabled?: boolean;
+    borderless?: boolean;
+    isActive?: boolean;
+    onClick?: () => void;
+    isProcessing?: boolean;
+    isCTA?: boolean;
+    icon?: ReactNode
 }
 
+const useStyles = createUseStyles((theme: any) => ({
```

Comment:
we already have an interface for theme (https://github.com/kaltura/path-design-system/pull/5/files#diff-13c5470b23d196f8b454b54cd36e2390), can we use it here instead of `any`?

Reply:
this is preferred, I will do so later because I want to merge this branch now so you will be able to continue your work.